### DISCUSSION TITLE
Cannot update user

### DISCUSSION THREAD

**Discussion by thefrana** [payloadcms/payload#866] [General]
# Bug Report

User data are not saved to the database.

## Expected Behavior

When I edit the user, entered data should be in the database.

## Current Behavior

When I edit the user, entered data are not in the database.

## Steps to Reproduce

I am unable to reproduce it. I have similar project and it does work in it.

```ts
import { CollectionConfig } from 'payload/types'

const Users: CollectionConfig = {
  slug: 'users',
  auth: true,
  admin: {
    useAsTitle: 'email',
  },
  access: {
    read: () => true,
  },
  fields: [
    {
      type: 'text',
      name: 'fullName',
      required: true,
    },
  ],
}

export default Users
```

This is my user. The field is shown, however the data from the input are not saved. The same goes for registrating the first user.

**Comment by thefrana** [comment]
Okay, I tried to dig into that. I deleted .cache in node_modules folder. It compiled .js files right next to the .ts ones. It should be compiled to dist folder.

I was then able to save the user. However, I could not save profile photo. It logged out me and broke login. When I manually deleted the profile photo field in the database, I logged in again without problems.

Then I saved the profile photo again without problems, but it saved ID as a string instead of ObjectId. It does not work as expected. It's all weird

**Comment by jmikrut** [comment]
Hey @lukeeno — can you give a bit more detail regarding exactly what you're trying to do, where it's failing, and what the error is that you're facing? 

When you are saying "could not save profile photo", are you referring to a separate document in a `Media` collection or similar? Or, are you trying to add a profile photo to a User document via a relationship field?

Document `id`s should be saved as an ObjectID, but when saving _relationship field_ data, we store related document IDs as strings. This is actually deliberate and will help us when we support additional databases in the future. Is that what you mean?

**Comment by jmikrut** [comment]
Hey @lukeeno — I'm gonna convert this into a discussion but if we can parse out specific things that are broken, I will open new issue(s) with the bugs that you find. But my hunch is this is just related to bad data in your database.

We are 100% around to help though - let's keep this going in the discussion!
